### PR TITLE
 [6] Product Page using HTML

### PR DESCRIPTION
**Pull request by Samju200**
## Task: `Develop a product page for your e-commerce store using HTML Only`
- `Add introduction to html folder`
[https://samju200.github.io/juwon/htmldufunatask/](https://marthaslivingbrands.com/)
html page 1
html page 2
html page 3
html page 4

### CODE REVIEW

File: submissions/adeyemiJuwon/introductionToHtml/index.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            <li> <a href="#"> 11 Inch Laptop Bag Tablet Zipper Pouch Sleeve for MacBook Air</a></li>
+            <li> <a href="#">SSIMO#O 2 in 1 Business Style Laptop Bag Tablet Zipper Pouch Sleeve for MacBook 13 / 14 inch</a></li>
+            <li> <a href="#">SSIMOO 2 in 1 Business Style Laptop Bag Tablet Zipper Pouch Sleeve for MacBook 15 inch</a></li>
+            <li> <a href="#">SSIMOO S818 2 in 1 Dot Pattern Laptop Bag Tablet Zipper Pouch Sleeve for MacBook 14 / 15 inch</a></li>
+        </ol>
+    </div>
+    <div>
+        <h1>SMART HOME ACCESSORIES</h1>
+        <a href="">
+            <img src="img/light.jpg" alt="">
```
Comment (ToluAkin) [inline submissions/adeyemiJuwon/introductionToHtml/index.html:64]: Well done.
You are meant to use 2 product images . Check the criteria here 
https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/issues/7

**Comment by Samju200** [comment]
> You should delete the images that you are not using.

Okay 
Done

**Comment by ToluAkin** [comment]
@Samju200 there are still 3 images added this pull request.

**Comment by Samju200** [comment]
> @Samju200 there are still 3 images added this pull request.

yes 
the logo
and the two product image

**Comment by Rac-72** [comment]
Hello

**Pull request merged**